src/main/java/fr/insalyon/creatis/gasw/parser/GaswParser.java

```diff
@@ -489,4 +495,56 @@ private static void addName(
             content.append(separator).append(f.getName());
         }
     }
-}
+
+    public GaswInput getGaswInput(String applicationName, Map<String, String> inputsMap, String executableName,HashMap<Integer, String> inputid , HashMap<Integer, String> outputid, String invocationString, 
```

Comment:
I think the final form of that should be pretty minimal and should be transferred in the gaswInput constructor.